app/components/ui/Navigation.tsx

```diff
@@ -34,8 +34,10 @@ const BURGER_STYLES = {
   },
 };
 
+// TODO: use @headlessui/react components for a11y
```

Comment:
we should be able to bring over what we did for Bookshare, at least in part, no?

Reply:
Yeah, I wish we wouldn't have done this fully custom